src/Styleguide/Elements/Select.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { Responsive } from "../Utils/Responsive"
+import { Sans } from "@artsy/palette"
+
+import { PositionProps, space, SpaceProps, themeGet } from "styled-system"
+
+export interface SelectProps extends PositionProps, SpaceProps {}
+
+export class Select extends React.Component<SelectProps> {
+  render() {
+    return (
+      <Responsive>
+        {({ xs }) => {
+          if (xs) return <LargeSelect />
+          else return <SmallSelect />
+        }}
+      </Responsive>
+    )
+  }
+}
+
+// Appears on both mobile and desktop
+export const LargeSelect = () => {
+  return (
+    <LargeSelectContainer p={3}>
+      <select>
+        <option value="0">Recently updated</option>
+        <option value="1">Recently added </option>
+        <option value="2">Artwork year (desc.)</option>
+        <option value="3">Artwork year (asc.)</option>
+      </select>
+    </LargeSelectContainer>
+  )
+}
+
+export const SmallSelect = () => {
+  return (
+    <SmallSelectContainer>
+      <Sans size="2" display="inline" mr={2}>
+        Sort:
```

Comment:
Seems like this would be the label field for whatever the select box is. I can't find the label replicated for the larger dropdown in the designs, but something like it is on production.

![image](https://user-images.githubusercontent.com/3087225/41269845-cb9d4ede-6dd5-11e8-82c3-3004963c9bae.png)

https://www.artsy.net/collect?sort=-decayed_merch